ttnn/cpp/ttnn/operations/data_movement/data_movement_pybind.hpp

```diff
@@ -78,6 +79,7 @@ void py_module(py::module& module) {
     py_bind_sharded_to_interleaved_partial(module);
     py_bind_squeeze(module);
     py_bind_unsqueeze(module);
+    expand::bind_expand_operation(module);
```

Comment:
Please align with existing style. You don't need the extra namespace.

Reply:
I've moved the pybind to `data_movement::detail`